src/SFA.DAS.ReferenceData.Organisations.Database/Scripts/PreDeployment/PreDeployment.sql

```diff
@@ -8,4 +8,11 @@
  Example:      :setvar TableName MyTable							
                SELECT * FROM [$(TableName)]					
 --------------------------------------------------------------------------------------
-*/
\ No newline at end of file
+*/
+ALTER TABLE [CharityImport].[extract_charity]  ALTER COLUMN name VARCHAR (500) NULL;
+ALTER TABLE [CharityImport].[extract_charity]  ALTER COLUMN orgtype VARCHAR (50) NULL;
+
+ALTER TABLE [CharityData].[charity]  ALTER COLUMN name VARCHAR (500) NULL;
+ALTER TABLE [CharityData].[charity]  ALTER COLUMN orgtype VARCHAR (50) NULL;
+
+ALTER TABLE [CharityData].[charitynamesearch]  ALTER COLUMN Name VARCHAR (500) NULL;
```

Comment:
These are unnecessary, they can simply be changed in the corresponding table .sql files in the DACPAC

Reply:
these don't seem to have been removed.